aws-ssm-document/src/main/java/com/amazonaws/ssm/document/tags/TagUtil.java

```diff
@@ -40,10 +46,20 @@ public static TagUtil getInstance() {
      */
     public boolean isResourceTagModified(@Nullable final List<Tag> previousResourceModelTags,
                                          @Nullable final List<Tag> currentResourceModelTags) {
-        return (previousResourceModelTags == null || previousResourceModelTags.isEmpty())
-                && (currentResourceModelTags == null || currentResourceModelTags.isEmpty());
+        if (previousResourceModelTags == null || previousResourceModelTags.isEmpty()) {
```

Comment:
nit: use `ObjectUtils` to simplify the not equal check.

Reply:
Ah... that's cool, I can add that in an update to this